sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/EventHubsClaim.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.EventHubs.Authorization
+{
+    /// <summary>
+    ///   The set of claims for operations associated with the Event Hubs service.
+    /// </summary>
+    ///
+    internal static class EventHubsClaim
+    {
+        /// <summary>Needed for management operations, such as querying metadata, against the service.</summary>
+        public const string Manage = "Manage";
+
+        /// <summary>Needed for sending data, such as publishing events, to the service.</summary>
+        public const string Send = "Send";
+
+        /// <summary>Needed for receiving data, such as events, from the service. </summary>
+        public const string Listen = "Listen";
```

Comment:
Curious that this isn't named `Receive = "Receive";` is that an Event Hubs thing? :)

Reply:
These are the claims that the service mapped in Azure; we don't get a vote on their name or meaning.  /shrug